extensions/amp-analytics/amp-analytics.md

```diff
@@ -557,3 +557,7 @@ If set to `include`, this turns on the ability to read and write cookies on the
 **data-consent-notification-id**
 
 If provided, the page will not process analytics requests until an [amp-user-notification](../../extensions/amp-user-notification/amp-user-notification.md) with the given HTML element id is confirmed (accepted) by the user. This is an optional attribute.
+
+
+## Analytics for AMP components
+Individual AMP component can implement with AMP analytics to send analytics data. For more information, please refer to [Implementing analytics for AMP components](./amp-components-analytics.md)
```

Comment:
AMP component developers can implement collection of data using AMP analytics. For more information ...